README.md

```diff
@@ -1,30 +1,103 @@
+FreeOrion requires an *OpenGL 2.1* capable graphic card and a display with a
+minimum resolution of at least *800x600*.
 
+FreeOrion requires at least *Windows XP* with *Service Pack 2* or later,
+*Mac OSX 10.7* or later or any reasonable recent Linux distribution on x86
+compatible processors.  Other operating systems and architectures have reported
+to be working by users, but are not actively supported by the FreeOrion project.
 
-## License
 
-Artistic or creative content is licensed under the Creative Commons Attribution-Share Alike 3.0 Unported license.
-(http://creativecommons.org/licenses/by-sa/3.0/)
+## Download
+
+[FreeOrion Stable Releases] are the recommend way to play FreeOrion.  Stable
+Releases can be obtained as native installer binaries for Windows and Mac OSX
+or as source releases for Linux and other UNIX-ish platforms from GitHub.
+
+Also some Linux distributions like Fedora, Debian and Arch do provide packages
+of FreeOrion.
+
+[FreeOrion Weekly Releases] are in-development releases intended for enthusiasts
+and testers, who want to track or contribute to the development.  Weekly
+Releases can be obtained as native installer binaries for Windows and Mac OSX
+from Sourceforge.
+
+
+## Install
+
+For Windows execute the native installer binary and follow the on-screen
+instructions of the installer to install FreeOrion.
+
+For Mac OSX, open the downloaded DMG file and copy the contained FreeOrion
+application to your system Applications folder by Drag and Drop.
+
+For Linux or other from-source installations in general please refer to the
+[Compile Instructions].
+
+Various Linux distributions provide the stable release of FreeOrion in
+a prebuilt way.  Usually you can install those packages by either using
+a graphical package manager and searching for *FreeOrion* or by installing the
+packages via the command line.
+
+  * [Debian package] stable release: `# apt-get install freeorion`
+  * [Ubuntu package] stable release: `# apt-get install freeorion`
+  * [Fedora package] stable release: `# dnf install freeorion`
+  * [ArchLinux package] stable release
+
+
+## Get Help/Contact
+
+Feel free to visit the [FreeOrion Homepage] or the [FreeOrion Forum].
+
+
+## Contribute
+
+The FreeOrion project encourages anybody to contribute to FreeOrion. For more
+details please see the [Contribution Guidelines](CONTRIBUTING.md).
+
+
+## License
 
-This includes artwork, music, sounds, text that determines how art, music or sound is played or displayed in game, game content such as technologies, buildings, specials and their associated in-game text, and the scripts that determine their in-game functionality.
+The FreeOrion source code is licensed under the terms of [GPL v2],
+game assets are licensed under the terms of [CC-BY-SA-3.0].  For more details
+please see the [License File](default/COPYING).
 
-Game content scripts are also licensed under the GNU General Public License, Version 2
```

Comment:
Retain mention of "game content scripts", as these are not source code, but are intended to be GPL licensed.